src/services/redisSessionStorage.ts

```diff
@@ -230,46 +230,22 @@ export default class RedisSessionStorage extends RedisStorageUtils
         `${sessionInfoKeyPrefix}${user.session_token}`
       );
     }
-    const userSessionTokensResult = await new Promise<ReadonlyArray<string>>(
-      (resolve, reject) => {
-        const keys = isRight(sessionKeys)
-          ? sessionKeys.value
-          : initializedSessionKeys;
-        this.redisClient.mget(...keys, (err, response) => {
-          if (err) {
-            reject(err);
-          }
-          resolve(response);
-        });
-      }
-    );
-    return right(
-      userSessionTokensResult.reduce(
-        (prev: SessionsList, _) => {
-          try {
-            const sessionInfoPayload = JSON.parse(_);
-            const errorOrDeserializedSessionInfo = SessionInfo.decode(
-              sessionInfoPayload
-            );
-
-            if (isLeft(errorOrDeserializedSessionInfo)) {
-              log.warn(
-                "Unable to decode the session info: %s. Skipped.",
-                ReadableReporter.report(errorOrDeserializedSessionInfo)
-              );
-              return prev;
-            }
-            return {
-              sessions: [...prev.sessions, errorOrDeserializedSessionInfo.value]
-            };
-          } catch (err) {
-            log.error("Unable to parse the session info json. Skipped.");
-            return prev;
-          }
-        },
-        { sessions: [] } as SessionsList
-      )
-    );
+    try {
+      const userSessionTokensResult = await new Promise<ReadonlyArray<string>>(
+        (resolve, reject) => {
+          const keys = isRight(sessionKeys)
```

Comment:
sessionKeys.getOrElse(initializedSessionKeys)

Reply:
Fixed in ab4bfa2